share/instruments/string_tunings_presets.json

```diff
@@ -0,0 +1,308 @@
+[
+  {
+    "familyOrInstrumentId": "guitars",
```

Comment:
What do you think about storing this as 2 separate properties - "family" and "instrumentId"? Thus, if "instrumentId" is not specified, we apply this preset to the entire family. Otherwise, we apply it to the instrument. This looks like a more flexible solution